packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,135 @@
+                Accordion two content
+            <div slot="heading">Accordion two</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+            Accordion three content<div slot="heading">Accordion three</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion four content
+            <div slot="heading">Accordion four</div>
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### API
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_ appearance (`block`, `outline`, `ghost`), `error-visible` (boolean)
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_ Unchanged
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ `error-visible` and accordion appearances (`block`, `outline`, `ghost`)
+
+    The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` buttons. The nimble-button pattern with these styles seems to fit the styling of the accordion, so these styles can be used in the heading css class. The attributes of `block`, `outline`, and `ghost` will most likely be implemented with the name `appearance`. A few styling elements like the background color change on click that is used in the button will have to be removed.
+
+    The Figma design also includes a state that is most likely `error`, as it has a red border and red exclamation mark icon. For this state, the styling of the accordion header button could be altered to have a red border. This could be controlled with a boolean attribute such as `error-visible`, which would have a default of "" in the accordion attributes- when necessary, "error-visible" would be added to the accordion attributes and this would make the accordion switch to the red styling. Adding the error icon could be done by putting the icon in the `end` slot of the accordion heading.
+
+    A css class for the nimble-accordion-item can be used to get a green border (or red for `error-visible`) around the entire accordion (parent and children of accordion).
```

Comment:
I don't think we'd put a class like this on the accordion item (if anything, it would be on the accordion), but regardless, this gets into a level of implementation detail that probably doesn't need to be in this spec.